### DISCUSSION TITLE
Snapper rollback and read-only file system

### DISCUSSION THREAD

**Discussion by ghost** [SpiralLinux/SpiralLinux-project#334] [Q&A] [answered]
I'm brand new to Spiral Linux. I thought I'd better try out a roll-back now to reassure myself it will work for me when I need it. I booted into an earlier snapshot, and have now found that my filesytem is mounting read-only. I've no understanding of the workings of snapper (intend to acquire!) but can someone help me out with this?

**Comment by geckolinux** [comment, marked as answer]
Hi there, please try these instructions:
https://github.com/SpiralLinux/SpiralLinux-project/wiki#working-with-btrfs-snapshots-and-rollbacks

**Comment by ghost** [comment]
Thanks, that's helpful and now clear. I've also found this Suse guide useful for this and other features of Snapper: 
[https://documentation.suse.com/sles/12-SP4/html/SLES-all/cha-snapper.html](url)
